test/Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration.Tests/ExtensionsTests/StringExtensionTest.cs

```diff
@@ -45,60 +45,94 @@ public static IEnumerable<object[]> GetTestCasesForToCamelCaseShouldSucceed()
             };
         }
 
-        public static IEnumerable<object[]> GetTestCasesForRemoveRoslynDuplicateStringShouldSucceed()
+        public static IEnumerable<object[]> GetTestCasesForRemoveRoslynDuplicateStringShouldUpdateCorrectly()
         {
+            yield return new object[]
+            {
+                "String with even length starting from -",
```

Comment:
The description is a bit off. The whole string is even, but what matters is the part `-ef-efg` which is odd.

Can we say "string with the special-character-containing part having odd length starting with -"?